auth-web/tests/unit/components/TransactionsDataTable.spec.ts

```diff
@@ -94,12 +95,13 @@ describe('TransactionsDataTable tests', () => {
     expect(row1Cells.at(0).find('span').text()).toBe('label1 value1')
     expect(row1Cells.at(1).text()).toBe('ab12')
     expect(row1Cells.at(2).text()).toBe('tester 123')
-    expect(row1Cells.at(3).text()).toContain('January 24, 20233:09 PM')
+    expect(row1Cells.at(3).text()).toContain('January 24, 20237:00 PM')
```

Comment:
you'll want to set this to 6:00 PM based on the setup value (its in january so the difference will be 8 hours)

Reply:
https://www.worldtimebuddy.com/

Reply:
yeah the time you are testing is in january not March so it is an 8 hour difference

Reply:
you shouldn't be commenting out that check -- it is what tests the time conversion

Reply:
sorry haha -- I see you updated it